MovingBlocks/Terasology
# Line class refactoring & unit tests

### Contains

- Extracts the coordinate computations from `Canvas#drawLine` into a separate helper method in the existing `Line` class and adds some unit tests for the new method.

### How to test

- Use the BT editor to check that line rendering is working properly.
- Verify that the new tests are not failing and that their expected/actual values are appropriate.

### Outstanding before merging

No outstanding issues.
